task9/repository/meteorsClient.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+import { config } from "../config/config.js";
+
+export const getMeteorData = async(startDate, endDate) => {
+    try {
```

Comment:
You can remove try..catch here and leave it only in the controller

Reply:
done